src/test/java/guitests/KeyboardShortcutsTest.java

```diff
@@ -169,10 +169,13 @@ public void keyboardShortcutsTest() {
         // test shortcut on repo dropdown
         doubleClick(repoSelectorComboBox);
         pushKeys(SWITCH_DEFAULT_REPO);
-        waitUntilNodeAppears("#dummy/dummy_col1_1");
+        // wait for issue 9 to appear then click on it
+        // issue 9 is chosen instead of issue 10
+        // as there is a problem with finding issue 10's node due to it being the first card in the panel
+        waitUntilNodeAppears("#dummy/dummy_col1_9");
```

Comment:
Yup, from how you tested, it looks like it's just a problem with selecting the first line.